src/main/java/me/youhavetrouble/justchat/JustChat.java

```diff
@@ -2,31 +2,25 @@
 
 import net.kyori.adventure.text.minimessage.MiniMessage;
 import org.bukkit.configuration.file.FileConfiguration;
+import org.bukkit.plugin.Plugin;
 import org.bukkit.plugin.java.JavaPlugin;
 
 public final class JustChat extends JavaPlugin {
 
-    private static String chatFormat;
+    public static Plugin plugin;
```

Comment:
- This should be `JustChat` class, not `Plugin` class
- This should not be public. public variables can be reassigned at any time. We want this to always be the same instance of the plugin. Set it to private and create a getter method instead